README.md

```
@@ -17,201 +17,22 @@ Other popular tools offering variant effect/consequence prediction include:
+To annotate variant consequences, gnomAD frequencies and clinVar information for sequence variants:
+```sh
+    mehari annotate seqvars \
+      --transcripts resources/transcript_db \
+      --frequencies resources/gnomad_db \
+      --clinvar resources/clinvar_db \
+      --path-input-vcf input.vcf \
+      --path-output-vcf output.vcf
+```
+The corresponding database builds can be obtained from:
+ - transcripts: https://github.com/varfish-org/mehari-data-tx/releases
+ - gnomAD frequencies: TODO
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)